lib/text.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Text
+
+  ACCOUNT_SID = 'AC5275f9aaa1c28fffbfc9ae66f5b49eb1'
+  AUTH_TOKEN = '8ed68af1be4a108b5201e1f55303e0d3'
+
+  def initialize
+    @client = Twilio::REST::Client.new(ACCOUNT_SID, AUTH_TOKEN)
+  end
+
+  def send_message
+    @client.messages.create(
+                               from: '+15005550006',
+                               body: 'Thank you! Your order was placed and will
+                               be delivered in 1 hour from now',
```

Comment:
ONE HOUR!
I'm ordering for collection next time!

Reply:
Just kidding!
I had the same wait time in mine, however i went a bit extra and called on the 
`Time(.class/module?)`
`body: "Thank you! Your order was placed and will be delivered before #{((Time.new.hour)+(1))}:#{(Time.new.min)}"`
give it a try!

Reply:
Yes I completely forgot about setting a time .... I have been really silly on this one... 
(tbh I did it at last step and my brain was just nor able to think anymore)